tests/cyclops/query/test_interface.py

```diff
@@ -41,10 +42,37 @@ def test_query_interface(
     with pytest.raises(ValueError):
         query_interface.save(path, file_format="donkey")
 
-    generator = query_interface.run_in_grouped_batches(id_col="donkey", batch_size=10)
-    assert isinstance(generator, Generator)
 
-    query_interface.save_in_grouped_batches("test_save", id_col="donkey", batch_size=10)
+@pytest.mark.integration_test
+def test_query_interface_integration():
+    """Test QueryInterface with OMOPQuerier."""
+    synthea = OMOPQuerier("cdm_synthea10", database="synthea_integration_test")
```

Comment:
You know this code base way better than me, but have you considered splitting this test up in smaller parts to limit the scope of what a single test is testing? Or would that just make maintaining them all annoying?